Classes/WPEditorToolbarButton.m

```diff
@@ -59,93 +65,86 @@ - (void)setupAnimations
 	[self addTarget:self action:@selector(touchDown:) forControlEvents:UIControlEventTouchDown];
 	[self addTarget:self action:@selector(touchDragInside:) forControlEvents:UIControlEventTouchDragInside];
 	[self addTarget:self action:@selector(touchDragOutside:) forControlEvents:UIControlEventTouchDragOutside];
+    
+    self.circleScaleAnimation = [CABasicAnimation animationWithKeyPath:@"transform.scale"];
+    self.circleScaleAnimation.fromValue = [NSNumber numberWithFloat:0.5f];
+    self.circleScaleAnimation.toValue = [NSNumber numberWithFloat:1.4f];
+    [self.circleScaleAnimation setTimingFunction:[CAMediaTimingFunction functionWithName:kCAMediaTimingFunctionEaseIn]];
+    
+    self.circleOpacityAnimation = [CABasicAnimation animationWithKeyPath:@"opacity"];
+    self.circleOpacityAnimation.fillMode = kCAFillModeForwards;
+    self.circleOpacityAnimation.toValue = [NSNumber numberWithFloat:0.0];
+    [self.circleOpacityAnimation setTimingFunction:[CAMediaTimingFunction functionWithName:kCAMediaTimingFunctionEaseIn]];
+}
+
+- (void)startAnimation
+{
+    CGFloat circleLineWidth = 1.0f;
+    CGRect circleRect = CGRectInset(self.bounds, circleLineWidth / 2, circleLineWidth / 2);
```

Comment:
For type consistency the denominators should probably be a `CGFloat` literals. Maybe just add a `.0`?